vnext/Chakra/ChakraExecutor.cpp

```diff
@@ -369,14 +369,16 @@ void ChakraExecutor::initOnJSVMThread()
   JsSetContextData(m_context, this);
 
 #if !defined(USE_EDGEMODE_JSRT)
-  JsValueRef debuggerConsoleObject;
-  tls_runtimeTracker.DebugProtocolHandler->GetConsoleObject(&debuggerConsoleObject);
+  if (enableDebugging) {
```

Comment:
The check for m_instanceArgs.DebuggerConsoleRedirection should be here. No need to get the debugger console object if this is not enabled.

Reply:
updated